### PR TITLE
Accept conversion of boundaries without errors in n/b networks

### PR DESCRIPTION
**Pull request by MioRtia**
Signed-off-by: RALAMBOTIANA MIORA <miora.ralambotiana@rte-france.com>

**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes / features)

**What kind of change does this PR introduce?** *(Bug fix, feature, docs update, ...)*
Add feature

**What is the current behavior?** *(You can also link to an open issue here)*
We can only convert boundaries for CGMES BusBranch networks

**What is the new behavior (if this is a feature change)?**
CGMES NodeBreaker networks can also be converted with boundaries

### CODE REVIEW

File: cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/elements/NodeConversion.java [hunk 1]
```diff
@@ -156,7 +159,14 @@ public void setVoltageAngleNodeBreaker() {
         if (!checkValidVoltageAngle(null)) {
             return;
         }
-        VoltageLevel vl = voltageLevel();
+        VoltageLevel vl;
+        if (insideBoundary() && context.config().convertBoundary()) {
```
Comment (zamarrenolm) [inline cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/elements/NodeConversion.java]: Maybe this logic for getting the right voltage level should go in the `voltageLevel()` function

File: cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/elements/NodeConversion.java [hunk 2]
```diff
@@ -174,14 +184,14 @@ public void setVoltageAngleNodeBreaker() {
         }
         Bus bus = t.getBusView().getBus();
         if (bus == null) {
-            bus = t.getBusBreakerView().getBus();
+            bus = t.getBusBreakerView().getConnectableBus();
```
Comment (zamarrenolm) [inline cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/elements/NodeConversion.java]: We are looking for a bus to set a voltage associated to this terminal. We must look for a connected bus in the IIDM network. If we obtain the connectable bus we are assigning the voltage to a different electric bus.

File: cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/elements/NodeConversion.java (hunk 1 above)
Comment (MioRtia) [reply cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/elements/NodeConversion.java]: Ok, done in the next commit

File: cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/elements/NodeConversion.java (hunk 2 above)
Comment (MioRtia) [reply cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/elements/NodeConversion.java]: Okay, I reverted the change. I did not pay attention to the fact that if the bus was disconnected, the connectable bus was calculated to get a different one.

**Pull request merged**